test-data-generator/pom.xml

```
@@ -59,4 +59,33 @@
         </dependency>
     </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <descriptorRefs>
+                        <descriptorRef>jar-with-dependencies</descriptorRef>
+                    </descriptorRefs>
+                    <appendAssemblyId>false</appendAssemblyId>
+                    <archive>
+                        <manifestEntries>
+                            <Main-Class>za.co.absa.spline.splinegen.App</Main-Class>
```

Package name is incorrect.
Also let's make the main class name consistent with the other main class names used in the project (AdminCLI)
```suggestion
                            <Main-Class>za.co.absa.spline.testdatagen.GenCLI</Main-Class>
```